clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -32,20 +32,126 @@ export const CustomTextInput = ({
   );
 };
 
+interface Option {
+  value: string;
+  label: string;
+}
+interface SelectProps {
+  label: string;
+  options: Option[];
+  isSearchable?: boolean;
+  isClearable?: boolean;
+}
 export const CustomSelect = ({
   label,
+  options,
+  isSearchable,
+  isClearable,
   ...props
-}: InputProps & FieldHookConfig<string>) => {
+}: SelectProps & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
+
+  const selected = options.find((o) => o.value === field.value);
+
+  return (
+    <FormControl isInvalid={isInvalid}>
+      <SimpleGrid columns={[1, 2]}>
+        <FormLabel htmlFor={props.id || props.name} size="sm">
+          {label}
+        </FormLabel>
+        <Select
+          options={options}
+          onBlur={(option) => {
+            if (option) {
+              field.onBlur(props.name);
+            }
+          }}
+          onChange={(newValue) => {
+            if (newValue) {
+              field.onChange(props.name)(newValue.value);
+            }
+          }}
+          name={props.name}
+          value={selected}
+          size="sm"
+          chakraStyles={{
```

Comment:
Are we providing a way to pass down styles to overwrite the styles set here? I notice not all the multiselect in the designs look the same.